addons/mail/models/mail_mail.py

```diff
@@ -50,6 +52,7 @@ class MailMail(models.Model):
     failure_reason = fields.Text(
         'Failure Reason', readonly=1,
         help="Failure reason. This is usually the exception thrown by the email server, stored to ease the debugging of mailing issues.")
+    scheduled_date = fields.Char('Scheduled Send Date')
```

Comment:
Add an help: If set, the queue manager will send the email after the date. If not set, the email will be send as soon as possible.